### DISCUSSION TITLE
!

### DISCUSSION THREAD

**Discussion by GoranSoce** [TanStack/table#4385] [Q&A]
!

**Comment by tannerlinsley** [comment]
I would have hoped you would spend more than 90 minutes to find a solution, but from your [other disruptive comment](https://github.com/TanStack/table/discussions/3525#discussioncomment-3655376)
> Sunk so much time to implement a table, and now I find out I can't set the fixed width of a column.

For others, I'll leave a copy of that response here:
> This is 100% possible and there are at least a few examples that show how to do this. In fact, even if you don't use the built-in sizing tools, you can always just put a fixed number in the column meta and manually apply it as a style in your table markup.

You can use the sizing api shown in a few of the examples, or as I suggested, do it manually.
